build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -xe
+
+# You can run it from any directory.
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PROJECT_DIR=$DIR/
+
+# This will: compile the project, run lint, run tests under JVM, package apk, check the code quality and run tests on the device/emulator.
+"$PROJECT_DIR"/gradlew --no-daemon --info clean
```

Comment:
Main trick is to split build steps to separate `gradle` invocations without daemonization